### PR TITLE
update readme with merge notes for homu delegates

### PR DESCRIPTION
**Pull request by metadave**
we can link to this when we delegating PR merge access

### CODE REVIEW

**Comment by thesentinels** [comment]
By analyzing the blame information on this pull request, we identified @juliandunn, @nathenharvey, @fnichol, @adamhjk and @smith to be potential reviewers

File: README.md [hunk 1]
```diff
@@ -217,6 +217,21 @@ routed to a [homu](https://github.com/barosl/homu) bot that will
...
+
+### Delegating pull request merge access
+
+A Habitat core maintainer can delegate pull request merge access to a contributor via 
+
+	@thesentinels delegate=username
+
+If you've been given approval to merge, you can do so by appending a comment to the pull request containing the following text:
+
+	@thesentinals r+
```
Comment (juliandunn) [inline README.md]: typo: thesentinals -> thesentinels

Comment (metadave) [inline README.md]: 😳 fixed

Comment (reset) [inline README.md]: Still got a typo on 229

Comment (metadave) [inline README.md]: I force pushed the change earlier, the PR comments still show the typo, but the commit has the fix (you can see it in the "Files Changes" tab)

**Comment by thesentinels** [comment]
:pushpin: Commit 0a673bd has been approved by `smith`

**Comment by thesentinels** [comment]
:hourglass: Testing commit 0a673bd with merge d2be9c1...

**Comment by thesentinels** [comment]
:sunny: Test successful - [travis](https://travis-ci.org/habitat-sh/habitat/builds/139827630)

**Pull request merged**